configs/default/IFRC-IFLIGHT_F411SX1280.config

```diff
@@ -0,0 +1,84 @@
+# Betaflight / STM32F411SX1280 (S4SX) 4.3.1 Jul 13 2022 / 03:33:32 (8d4f005) MSP API: 1.44
+
+board_name IFLIGHT_F411SX1280
```

Comment:
I would even prefer the shorter name `F4SX1280` as it's the only unified target with SX12x0

Reply:
Ok updated to F4SX1280